six/modules/c++/six/tests/test_polarizationType_conversions.cpp

```diff
@@ -0,0 +1,49 @@
+#include <six/Enums.h>
+#include <six/Utilities.h>
```

Comment:
Actually make this a unit test (under six/unittests) and use `TestCase.h` to do these tests